Update index.md

TwoMoreThings opened the pull request:
Added note about ensuring you inherit from SurfaceController and added the name to use for the partial view
Also changed wording around need to reload macros as this will hopefully be fixed in future

sofietoft commented on Getting-Started/Code/Creating-Forms/index.md:
```diff
@@ -37,6 +37,8 @@ Build your solution after adding the model.
 ### Creating the view
 Next, we add the view for the form to the `/View/Partials` folder. Because we've added the model and built the solution we can add it as a strongly typed view.
 
+Name your view "ContactForm"
```
```suggestion
Name your view "ContactForm".
```

sofietoft commented on Getting-Started/Code/Creating-Forms/index.md:
````diff
@@ -86,6 +88,8 @@ namespace MyFirstForm.Controllers
 }
 ```
 
+You should note that the controller inherits "SurfaceController" and not from "Controller" as initially added to the file by Visual Studio
````
```suggestion
You should note that the controller inherits "SurfaceController" and not from "Controller" as initially added to the file by Visual Studio.
```

sofietoft commented on Getting-Started/Code/Creating-Forms/index.md:
```diff
@@ -119,7 +123,9 @@ In the partial view, we're going to render our contact form using the view model
-The last thing to do before we can add the form to a page is to **allow the Macro in a rich text editor**. Right click __Macros__ and select __Reload__ then expand the __Macros__ node and select the __Contact Form__ Macro. Check the boxes under __Editor Settings__.
+The last thing to do before we can add the form to a page is to **allow the Macro in a rich text editor**. Expand the __Macros__ node and select the __Contact Form__ Macro. Check the boxes under __Editor Settings__.
+
+Tip: If you don't see your new macro listed, right click __Macros__ and select __Reload__
```
Excellent idea putting this into a tip here! 👏 

Have you seen the formatting options we have for tips like these?
You can see them here: https://our.umbraco.com/documentation/Contribute/Markdown-Conventions/#adding-notes-warnings-tips 
Makes the tip stands out a bit more! What do you think?

TwoMoreThings commented:
Thanks Sophie,
that's great
regards
t

On Thu, Oct 17, 2019 at 1:58 PM sofietoft <notifications@github.com> wrote:

> *@sofietoft* commented on this pull request.
>
> Again, great work @TwoMoreThings <https://github.com/TwoMoreThings> ! 😄
>
> I've added a couple of comments.
> ------------------------------
>
> In Getting-Started/Code/Creating-Forms/index.md
> <https://github.com/umbraco/UmbracoDocs/pull/2008#discussion_r335986061>:
>
> > @@ -37,6 +37,8 @@ Build your solution after adding the model.
>
>  ### Creating the view
>
>  Next, we add the view for the form to the `/View/Partials` folder. Because we've added the model and built the solution we can add it as a strongly typed view.
>
>
>
> +Name your view "ContactForm"
>
>
> ⬇️ Suggested change
>
> -Name your view "ContactForm"
>
> +Name your view "ContactForm".
>
>
> ------------------------------
>
> In Getting-Started/Code/Creating-Forms/index.md
> <https://github.com/umbraco/UmbracoDocs/pull/2008#discussion_r335986104>:
>
> > @@ -86,6 +88,8 @@ namespace MyFirstForm.Controllers
>
>  }
>
>  ```
>
>
>
> +You should note that the controller inherits "SurfaceController" and not from "Controller" as initially added to the file by Visual Studio
>
>
> ⬇️ Suggested change
>
> -You should note that the controller inherits "SurfaceController" and not from "Controller" as initially added to the file by Visual Studio
>
> +You should note that the controller inherits "SurfaceController" and not from "Controller" as initially added to the file by Visual Studio.
>
>
> ------------------------------
>
> In Getting-Started/Code/Creating-Forms/index.md
> <https://github.com/umbraco/UmbracoDocs/pull/2008#discussion_r335987472>:
>
> > @@ -119,7 +123,9 @@ In the partial view, we're going to render our contact form using the view model
>
>
>
>
>
>  #### Adding the macro
>
> -The last thing to do before we can add the form to a page is to **allow the Macro in a rich text editor**. Right click __Macros__ and select __Reload__ then expand the __Macros__ node and select the __Contact Form__ Macro. Check the boxes under __Editor Settings__.
>
> +The last thing to do before we can add the form to a page is to **allow the Macro in a rich text editor**. Expand the __Macros__ node and select the __Contact Form__ Macro. Check the boxes under __Editor Settings__.
>
> +
>
> +Tip: If you don't see your new macro listed, right click __Macros__ and select __Reload__
>
>
> Excellent idea putting this into a tip here! 👏
>
> Have you seen the formatting options we have for tips like these?
> You can see them here:
> https://our.umbraco.com/documentation/Contribute/Markdown-Conventions/#adding-notes-warnings-tips
> Makes the tip stands out a bit more! What do you think?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/umbraco/UmbracoDocs/pull/2008?email_source=notifications&email_token=ADKMY6ZCENM4KWBS5SJCPH3QPBONRA5CNFSM4JBYYKQKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCIJPF4I#pullrequestreview-303231729>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADKMY62DYWOKG2DCBBT2WCDQPBONRANCNFSM4JBYYKQA>
> .
>

sofietoft commented:
Awesome! 😄 
Would you mind committing the suggestions?
I don't have permissions to do that on your patch 🙈

sofietoft commented:
Thanks again @TwoMoreThings 😄 🎉
